src/auth/token-verifier.ts

```diff
@@ -209,7 +202,7 @@ export class FirebaseTokenVerifier {
         verifyJwtTokenDocsMessage;
     } else if (payload.iss !== this.issuer + projectId) {
       errorMessage = `${this.tokenInfo.jwtName} has incorrect "iss" (issuer) claim. Expected ` +
-        `"${this.issuer}"` + projectId + '" but got "' +
+        `"${this.issuer}` + projectId + '" but got "' +
```

Comment:
nit: took me a while staring at this diff because of the mixed use of string templates and string concatenation via `+`.   No need to do anything though, probably not in scope.